README.md

```diff
@@ -161,6 +161,12 @@ To specify an absolute path, add an additional forward slash to the path. The fo
 DATABASE_URL="sqlite:////tmp/database_name.sqlite3"
 ```
 
+**ClickHouse**
+
+```sh
+DATABASE_URL="clickhouse://127.0.0.1:9000?username=username&password=password&database=database_name"
```

Comment:
It's a bit silly that the clickhouse-go driver doesn't use the username/password fields already supported in a standard URL format (`username:password@127.0.0.1:9000`). Would be nice to support these as well in order to be consistent with other dbmate drivers.

Reply:
Actually, it's also weird that they expect the `database` as a URL parameter. The official [clickhouse jdbc driver](https://github.com/ClickHouse/clickhouse-jdbc) even uses a more standard format.

Could you add a bit of extra URL parsing (similar to what I do in the mysql driver) to support a standard url format like:

```
clickhouse://username:password@host:port/dabase_name
```

Under the hood you can convert these to the format `clickhouse-go` expects, and you could still support these query parameters if they are passed (just leave them unchanged) for interoperability? That will allow us to follow dbmate standards while also interop with clickhouse-go.

Reply:
Added parsing and a link to other CH-related connection string options.